src/main/webapp/i18n/de/course.json

```diff
@@ -228,7 +228,11 @@
             "activeStudents": "Aktive Studierende: {{ students }}",
             "notStartedYet": "Der Kurs hat noch nicht offiziell begonnen. Offizieller Kursstart:",
             "noOrganizations": "Keine Organisationen",
-            "organizationAddingHint": "Administratoren können Organisationen zu Kursen hinzufügen."
+            "organizationAddingHint": "Administratoren können Organisationen zu Kursen hinzufügen.",
+            "imageCropper": {
+                "title": "Bild zuchneiden",
+                "uploadImageClass": "no-image de"
```

Comment:
Remove this after you changed from image to HTML render

Reply:
Changed